util/git/client.go

```diff
@@ -424,7 +424,7 @@ func (m *nativeGitClient) Checkout(revision string, submoduleEnabled bool) error
 			}
 		}
 	}
-	if _, err := m.runCmd("clean", "-fdx"); err != nil {
+	if _, err := m.runCmd("clean", "-ffdx"); err != nil {
```

Comment:
Could you add a comment about the double `f`, just so that it's not mistaken for a typo?

Something like (from the `git clean` docs):

>  If the Git configuration variable clean.requireForce is not set to false, git clean will refuse to delete files or directories unless given -f or -i. Git will refuse to modify untracked nested git repositories (directories with a .git subdirectory) unless a second -f is given.

Reply:
Sure!